roles/kubernetes/node/defaults/main.yml

```diff
@@ -5,6 +5,12 @@ kubelet_deployment_type: host
 # change to 0.0.0.0 to enable insecure access from anywhere (not recommended)
 kube_apiserver_insecure_bind_address: 127.0.0.1
 
+# advertised host IP for kubelet. This affects network plugin config. Take caution
+kubelet_address: "{{ ip | default(ansible_default_ipv4['address']) }}"
+
+# bind address for kubelet. Set to 0.0.0.0 to listen on all interfaces
+kubelet_bind_address: "{{ ip | default(0.0.0.0) }}"
```

Comment:
I believe this line needs to be `"{{ ip | default('0.0.0.0') }}"` to avoid CI failures.

Reply:
you're right. I wasn't working very carefully